### PR TITLE
Fixed #962: Wrong title when moving from a fragment to MainActivity

### LINKED ISSUE #962: Issue with title when moving from a fragment to MainActivity
When going back from Fragment to MainActivity , the title of the SupportActionBar does not change

title1

The title still remains as Allergen alert
titlle2

### PR DESCRIPTION
**Pull request by PrajwalM2212**
## Description

When going back from Fragment to MainActivity , the title of the SupportActionBar now changes

## Related issues and discussion
#962 
 
 ## Screen-shots, if any
 
title1

The title now changes 

title2

### CODE REVIEW

**Comment by PrajwalM2212** [comment]
Will make the change . Thanks for the suggestion .

**Comment by PrajwalM2212** [comment]
@Karljoones I have made the changes.

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/FindProductFragment.java [hunk 1]
```diff
@@ -49,4 +51,26 @@ protected void onSearchBarcodeProduct() {
...
     }
+
+    public void onResume() {
+
+        super.onResume();
+
+        AppCompatActivity activity = (AppCompatActivity)getActivity();
+
+        ActionBar actionBar;
+
```
Comment (Karljoones) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/FindProductFragment.java]: Still a bit of code left over here from the old code, could you remove this?

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AlertUserFragment.java [hunk 2]
```diff
@@ -163,4 +165,22 @@ public static Integer getKey(HashMap<Integer, String> map, String value) {
         return key;
     }
 
+    public void onResume() {
+
+        super.onResume();
+
+        try {
+
+
```
Comment (Karljoones) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AlertUserFragment.java]: Could we also cut down on the amount of unnecessary whitespace here?

**Comment by PrajwalM2212** [comment]
@Karljoones Changes made

**Comment by teolemon** [comment]
Merging thanks @PrajwalM2212

**Pull request merged**